spec/models/defence_request_spec.rb

```diff
@@ -18,4 +19,11 @@
     it { expect(subject).to allow_value('020 1234 5678', '+44(0)7891234567').for(:phone_number) }
     it { expect(subject).to_not allow_value('1234', 'ABC').for(:phone_number) }
   end
+
+  describe :states do
+    specify { expect(subject.state).to eql 'open' }
+
+    specify { expect{ subject.close }.to_not raise_error }
+    specify { expect{ subject.close; subject.close }.to raise_error }
```

Comment:
Do we want to specify at least an error class here, otherwise it could pass when something is wrong